### ISSUE TITLE
[BUG] Many duplicates are generated from one E-Mail

### ISSUE THREAD

**Issue by highTowerSU** [paperless-ngx/paperless-ngx#5818]
### Description

Hello,

i have updated paperless and after this i scanned many documents in multiple E-Mails, some times seperated by "PATCH T" pages.

The result is, that they are imported many times.

![grafik](https://github.com/paperless-ngx/paperless-ngx/assets/3651850/ee174557-a504-4e2b-bea2-455983cd6942)

I have searched for bugs and other solutions but didn't find one.

### Steps to reproduce

Scan a document.
Use PATCH T Seperator Pages.
Wait on import.
See many duplicates. Some times alternating, some times 5 after the first.

### Webserver logs

```bash
[2024-02-19 06:54:27,369] [INFO] [paperless.consumer] Consuming image.pdf

[2024-02-19 06:54:27,376] [DEBUG] [paperless.consumer] Detected mime type: application/pdf

[2024-02-19 06:54:27,387] [DEBUG] [paperless.consumer] Parser: RasterisedDocumentParser

[2024-02-19 06:54:27,391] [DEBUG] [paperless.consumer] Parsing image.pdf...

[2024-02-19 06:54:27,643] [DEBUG] [paperless.parsing.tesseract] Calling OCRmyPDF with args: {'input_file': PosixPath('/tmp/paperless/paperless-ngxsftkf5e_/image.pdf'), 'output_file': PosixPath('/tmp/paperless/paperless-zgjdjujm/archive.pdf'), 'use_threads': True, 'jobs': 4, 'language': 'deu', 'output_type': 'pdfa', 'progress_bar': False, 'color_conversion_strategy': 'RGB', 'redo_ocr': True, 'clean': True, 'rotate_pages': True, 'rotate_pages_threshold': 12.0, 'sidecar': PosixPath('/tmp/paperless/paperless-zgjdjujm/sidecar.txt')}

[2024-02-19 06:55:11,865] [DEBUG] [paperless.consumer] Generating thumbnail for image.pdf...

[2024-02-19 06:55:11,872] [DEBUG] [paperless.parsing] Execute: convert -density 300 -scale 500x5000> -alpha remove -strip -auto-orient -define pdf:use-cropbox=true /tmp/paperless/paperless-zgjdjujm/archive.pdf[0] /tmp/paperless/paperless-zgjdjujm/convert.webp

[2024-02-19 06:55:14,519] [DEBUG] [paperless.consumer] Saving record to database

[2024-02-19 06:55:14,520] [DEBUG] [paperless.consumer] Creation date from parse_date: 2023-05-02 00:00:00+02:00

[2024-02-19 06:55:14,960] [INFO] [paperless.handlers] Assigning correspondent drv to 2023-05-02 iJames Internet-Fax

[2024-02-19 06:55:14,982] [INFO] [paperless.handlers] Assigning document type Bescheinigung to 2023-05-02 drv iJames Internet-Fax

[2024-02-19 06:55:15,107] [DEBUG] [paperless.consumer] Deleting file /tmp/paperless/paperless-ngxsftkf5e_/image.pdf

[2024-02-19 06:55:15,111] [DEBUG] [paperless.parsing.tesseract] Deleting directory /tmp/paperless/paperless-zgjdjujm

[2024-02-19 06:55:15,113] [INFO] [paperless.consumer] Document 2023-05-02 drv iJames Internet-Fax consumption finished

[2024-02-19 06:55:16,406] [DEBUG] [paperless.tasks] Skipping plugin CollatePlugin

[2024-02-19 06:55:16,407] [DEBUG] [paperless.tasks] Executing plugin BarcodePlugin

[2024-02-19 06:55:16,407] [DEBUG] [paperless.barcodes] Scanning for barcodes using PYZBAR

[2024-02-19 06:55:17,600] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 06:55:19,463] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 06:55:21,545] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 06:55:23,413] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 06:55:25,257] [DEBUG] [paperless.barcodes] Scanning for barcodes using PYZBAR

[2024-02-19 06:55:26,538] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 06:55:28,248] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 06:55:30,329] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 06:55:32,142] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 06:55:33,851] [DEBUG] [paperless.barcodes] Scanning for barcodes using PYZBAR

[2024-02-19 06:55:35,080] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 06:55:36,866] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 06:55:38,907] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 06:55:40,834] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 06:55:42,648] [INFO] [paperless.tasks] BarcodePlugin completed with no message

[2024-02-19 06:55:42,703] [DEBUG] [paperless.tasks] Executing plugin WorkflowTriggerPlugin

[2024-02-19 06:55:42,709] [INFO] [paperless.tasks] WorkflowTriggerPlugin completed with no message

[2024-02-19 06:55:42,736] [INFO] [paperless.consumer] Consuming image.pdf

[2024-02-19 06:55:42,742] [DEBUG] [paperless.consumer] Detected mime type: application/pdf

[2024-02-19 06:55:42,754] [DEBUG] [paperless.consumer] Parser: RasterisedDocumentParser

[2024-02-19 06:55:42,758] [DEBUG] [paperless.consumer] Parsing image.pdf...

[2024-02-19 06:55:43,004] [DEBUG] [paperless.parsing.tesseract] Calling OCRmyPDF with args: {'input_file': PosixPath('/tmp/paperless/paperless-ngxyve1vdz_/image.pdf'), 'output_file': PosixPath('/tmp/paperless/paperless-teowi3_c/archive.pdf'), 'use_threads': True, 'jobs': 4, 'language': 'deu', 'output_type': 'pdfa', 'progress_bar': False, 'color_conversion_strategy': 'RGB', 'redo_ocr': True, 'clean': True, 'rotate_pages': True, 'rotate_pages_threshold': 12.0, 'sidecar': PosixPath('/tmp/paperless/paperless-teowi3_c/sidecar.txt')}

[2024-02-19 06:56:07,257] [DEBUG] [paperless.consumer] Generating thumbnail for image.pdf...

[2024-02-19 06:56:07,264] [DEBUG] [paperless.parsing] Execute: convert -density 300 -scale 500x5000> -alpha remove -strip -auto-orient -define pdf:use-cropbox=true /tmp/paperless/paperless-teowi3_c/archive.pdf[0] /tmp/paperless/paperless-teowi3_c/convert.webp

[2024-02-19 06:56:11,816] [DEBUG] [paperless.consumer] Saving record to database

[2024-02-19 06:56:11,817] [DEBUG] [paperless.consumer] Creation date from parse_date: 2023-08-26 00:00:00+02:00

[2024-02-19 06:56:12,364] [INFO] [paperless.handlers] Assigning correspondent juh to 2023-08-26 iJames Internet-Fax

[2024-02-19 06:56:12,376] [INFO] [paperless.handlers] Assigning document type Schreiben to 2023-08-26 juh iJames Internet-Fax

[2024-02-19 06:56:15,543] [DEBUG] [paperless.consumer] Deleting file /tmp/paperless/paperless-ngxyve1vdz_/image.pdf

[2024-02-19 06:56:15,551] [DEBUG] [paperless.parsing.tesseract] Deleting directory /tmp/paperless/paperless-teowi3_c

[2024-02-19 06:56:15,553] [INFO] [paperless.consumer] Document 2023-08-26 juh iJames Internet-Fax consumption finished

[2024-02-19 06:56:15,960] [DEBUG] [paperless.tasks] Skipping plugin CollatePlugin

[2024-02-19 06:56:15,960] [DEBUG] [paperless.tasks] Executing plugin BarcodePlugin

[2024-02-19 06:56:15,961] [DEBUG] [paperless.barcodes] Scanning for barcodes using PYZBAR

[2024-02-19 06:56:16,379] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 06:56:18,745] [DEBUG] [paperless.barcodes] Barcode of type QRCODE found: BEGIN:VEVENT

...

END:VEVENT

[2024-02-19 06:56:18,747] [DEBUG] [paperless.barcodes] Barcode of type QRCODE found: ...


[2024-02-19 06:56:18,760] [INFO] [paperless.tasks] BarcodePlugin completed with no message

[2024-02-19 06:56:18,767] [DEBUG] [paperless.tasks] Executing plugin WorkflowTriggerPlugin

[2024-02-19 06:56:18,770] [INFO] [paperless.tasks] WorkflowTriggerPlugin completed with no message

[2024-02-19 06:56:18,789] [INFO] [paperless.consumer] Consuming image.pdf

[2024-02-19 06:56:18,793] [DEBUG] [paperless.consumer] Detected mime type: application/pdf

[2024-02-19 06:56:18,801] [DEBUG] [paperless.consumer] Parser: RasterisedDocumentParser

[2024-02-19 06:56:18,805] [DEBUG] [paperless.consumer] Parsing image.pdf...

[2024-02-19 06:56:19,027] [DEBUG] [paperless.parsing.tesseract] Calling OCRmyPDF with args: {'input_file': PosixPath('/tmp/paperless/paperless-ngx6oejayb7/image.pdf'), 'output_file': PosixPath('/tmp/paperless/paperless-leh4307b/archive.pdf'), 'use_threads': True, 'jobs': 4, 'language': 'deu', 'output_type': 'pdfa', 'progress_bar': False, 'color_conversion_strategy': 'RGB', 'redo_ocr': True, 'clean': True, 'rotate_pages': True, 'rotate_pages_threshold': 12.0, 'sidecar': PosixPath('/tmp/paperless/paperless-leh4307b/sidecar.txt')}

[2024-02-19 06:56:40,240] [DEBUG] [paperless.consumer] Generating thumbnail for image.pdf...

[2024-02-19 06:56:40,251] [DEBUG] [paperless.parsing] Execute: convert -density 300 -scale 500x5000> -alpha remove -strip -auto-orient -define pdf:use-cropbox=true /tmp/paperless/paperless-leh4307b/archive.pdf[0] /tmp/paperless/paperless-leh4307b/convert.webp

[2024-02-19 06:56:43,872] [DEBUG] [paperless.consumer] Saving record to database

[2024-02-19 06:56:43,873] [DEBUG] [paperless.consumer] Creation date from parse_date: 2023-05-05 00:00:00+02:00

[2024-02-19 06:56:44,316] [INFO] [paperless.handlers] Assigning correspondent juh to 2023-05-05 iJames Internet-Fax

[2024-02-19 06:56:44,324] [INFO] [paperless.handlers] Assigning document type Bescheinigung to 2023-05-05 juh iJames Internet-Fax

[2024-02-19 06:56:44,403] [DEBUG] [paperless.consumer] Deleting file /tmp/paperless/paperless-ngx6oejayb7/image.pdf

[2024-02-19 06:56:44,410] [DEBUG] [paperless.parsing.tesseract] Deleting directory /tmp/paperless/paperless-leh4307b

[2024-02-19 06:56:44,412] [INFO] [paperless.consumer] Document 2023-05-05 juh iJames Internet-Fax consumption finished

[2024-02-19 06:56:44,786] [DEBUG] [paperless.tasks] Skipping plugin CollatePlugin

[2024-02-19 06:56:44,786] [DEBUG] [paperless.tasks] Executing plugin BarcodePlugin

[2024-02-19 06:56:44,787] [DEBUG] [paperless.barcodes] Scanning for barcodes using PYZBAR

[2024-02-19 06:56:45,141] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 06:56:46,999] [DEBUG] [paperless.barcodes] Scanning for barcodes using PYZBAR

[2024-02-19 06:56:47,347] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 06:56:49,091] [DEBUG] [paperless.barcodes] Scanning for barcodes using PYZBAR

[2024-02-19 06:56:49,439] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 06:56:51,169] [INFO] [paperless.tasks] BarcodePlugin completed with no message

[2024-02-19 06:56:51,183] [DEBUG] [paperless.tasks] Executing plugin WorkflowTriggerPlugin

[2024-02-19 06:56:51,187] [INFO] [paperless.tasks] WorkflowTriggerPlugin completed with no message

[2024-02-19 06:56:51,203] [INFO] [paperless.consumer] Consuming image.pdf

[2024-02-19 06:56:51,207] [DEBUG] [paperless.consumer] Detected mime type: application/pdf

[2024-02-19 06:56:51,215] [DEBUG] [paperless.consumer] Parser: RasterisedDocumentParser

[2024-02-19 06:56:51,219] [DEBUG] [paperless.consumer] Parsing image.pdf...

[2024-02-19 06:56:51,438] [DEBUG] [paperless.parsing.tesseract] Calling OCRmyPDF with args: {'input_file': PosixPath('/tmp/paperless/paperless-ngxxub4afp0/image.pdf'), 'output_file': PosixPath('/tmp/paperless/paperless-gjxwlijs/archive.pdf'), 'use_threads': True, 'jobs': 4, 'language': 'deu', 'output_type': 'pdfa', 'progress_bar': False, 'color_conversion_strategy': 'RGB', 'redo_ocr': True, 'clean': True, 'rotate_pages': True, 'rotate_pages_threshold': 12.0, 'sidecar': PosixPath('/tmp/paperless/paperless-gjxwlijs/sidecar.txt')}

[2024-02-19 06:57:09,912] [DEBUG] [paperless.consumer] Generating thumbnail for image.pdf...

[2024-02-19 06:57:09,917] [DEBUG] [paperless.parsing] Execute: convert -density 300 -scale 500x5000> -alpha remove -strip -auto-orient -define pdf:use-cropbox=true /tmp/paperless/paperless-gjxwlijs/archive.pdf[0] /tmp/paperless/paperless-gjxwlijs/convert.webp

[2024-02-19 06:57:15,820] [DEBUG] [paperless.consumer] Saving record to database

[2024-02-19 06:57:15,821] [DEBUG] [paperless.consumer] Creation date from st_mtime: 2024-02-19 05:27:30.583834+01:00

[2024-02-19 06:57:16,406] [INFO] [paperless.handlers] Assigning correspondent juh to 2024-02-19 iJames Internet-Fax

[2024-02-19 06:57:16,416] [INFO] [paperless.handlers] Assigning document type Schreiben to 2024-02-19 juh iJames Internet-Fax

[2024-02-19 06:57:16,504] [DEBUG] [paperless.consumer] Deleting file /tmp/paperless/paperless-ngxxub4afp0/image.pdf

[2024-02-19 06:57:16,512] [DEBUG] [paperless.parsing.tesseract] Deleting directory /tmp/paperless/paperless-gjxwlijs

[2024-02-19 06:57:16,514] [INFO] [paperless.consumer] Document 2024-02-19 juh iJames Internet-Fax consumption finished

[2024-02-19 06:57:17,722] [DEBUG] [paperless.tasks] Skipping plugin CollatePlugin

[2024-02-19 06:57:17,722] [DEBUG] [paperless.tasks] Executing plugin BarcodePlugin

[2024-02-19 06:57:17,723] [DEBUG] [paperless.barcodes] Scanning for barcodes using PYZBAR

[2024-02-19 06:57:18,155] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 06:57:20,073] [DEBUG] [paperless.barcodes] Scanning for barcodes using PYZBAR

[2024-02-19 06:57:20,436] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 06:57:22,129] [DEBUG] [paperless.barcodes] Scanning for barcodes using PYZBAR

[2024-02-19 06:57:22,487] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 06:57:24,207] [INFO] [paperless.tasks] BarcodePlugin completed with no message

[2024-02-19 06:57:24,222] [DEBUG] [paperless.tasks] Executing plugin WorkflowTriggerPlugin

[2024-02-19 06:57:24,225] [INFO] [paperless.tasks] WorkflowTriggerPlugin completed with no message

[2024-02-19 06:57:24,242] [INFO] [paperless.consumer] Consuming image.pdf

[2024-02-19 06:57:24,245] [DEBUG] [paperless.consumer] Detected mime type: application/pdf

[2024-02-19 06:57:24,255] [DEBUG] [paperless.consumer] Parser: RasterisedDocumentParser

[2024-02-19 06:57:24,259] [DEBUG] [paperless.consumer] Parsing image.pdf...

[2024-02-19 06:57:24,474] [DEBUG] [paperless.parsing.tesseract] Calling OCRmyPDF with args: {'input_file': PosixPath('/tmp/paperless/paperless-ngx7nzi57cr/image.pdf'), 'output_file': PosixPath('/tmp/paperless/paperless-hxtihwtm/archive.pdf'), 'use_threads': True, 'jobs': 4, 'language': 'deu', 'output_type': 'pdfa', 'progress_bar': False, 'color_conversion_strategy': 'RGB', 'redo_ocr': True, 'clean': True, 'rotate_pages': True, 'rotate_pages_threshold': 12.0, 'sidecar': PosixPath('/tmp/paperless/paperless-hxtihwtm/sidecar.txt')}

[2024-02-19 06:57:40,318] [DEBUG] [paperless.consumer] Generating thumbnail for image.pdf...

[2024-02-19 06:57:40,325] [DEBUG] [paperless.parsing] Execute: convert -density 300 -scale 500x5000> -alpha remove -strip -auto-orient -define pdf:use-cropbox=true /tmp/paperless/paperless-hxtihwtm/archive.pdf[0] /tmp/paperless/paperless-hxtihwtm/convert.webp

[2024-02-19 06:57:46,291] [DEBUG] [paperless.consumer] Saving record to database

[2024-02-19 06:57:46,292] [DEBUG] [paperless.consumer] Creation date from st_mtime: 2024-02-19 05:27:30.843833+01:00

[2024-02-19 06:57:46,878] [INFO] [paperless.handlers] Assigning correspondent bb to 2024-02-19 iJames Internet-Fax

[2024-02-19 06:57:46,887] [INFO] [paperless.handlers] Assigning document type Bescheid to 2024-02-19 bb iJames Internet-Fax

[2024-02-19 06:57:46,976] [DEBUG] [paperless.consumer] Deleting file /tmp/paperless/paperless-ngx7nzi57cr/image.pdf

[2024-02-19 06:57:46,980] [DEBUG] [paperless.parsing.tesseract] Deleting directory /tmp/paperless/paperless-hxtihwtm

[2024-02-19 06:57:46,982] [INFO] [paperless.consumer] Document 2024-02-19 bb iJames Internet-Fax consumption finished

[2024-02-19 06:57:47,377] [DEBUG] [paperless.tasks] Skipping plugin CollatePlugin

[2024-02-19 06:57:47,377] [DEBUG] [paperless.tasks] Executing plugin BarcodePlugin

[2024-02-19 06:57:47,378] [DEBUG] [paperless.barcodes] Scanning for barcodes using PYZBAR

[2024-02-19 06:57:52,561] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 06:57:54,702] [DEBUG] [paperless.barcodes] Barcode of type QRCODE found: https://www.instagram.com/abc

[2024-02-19 06:57:54,703] [DEBUG] [paperless.barcodes] Barcode of type QRCODE found: https://www.facebook.com/abc

[2024-02-19 06:57:54,706] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 06:57:56,512] [DEBUG] [paperless.barcodes] Barcode of type QRCODE found: https://www.facebook.com/abc

[2024-02-19 06:57:56,512] [DEBUG] [paperless.barcodes] Barcode of type QRCODE found: https://www.instagram.com/abc

[2024-02-19 06:57:56,514] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 06:57:58,416] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 06:58:00,141] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 06:58:01,742] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 06:58:03,304] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 06:58:04,770] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 06:58:06,724] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 06:58:08,421] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 06:58:10,141] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 06:58:11,750] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 06:58:13,628] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 06:58:15,396] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 06:58:17,181] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 06:58:18,871] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 06:58:20,594] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 06:58:22,506] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 06:58:24,289] [INFO] [paperless.tasks] BarcodePlugin completed with no message

[2024-02-19 06:58:24,334] [DEBUG] [paperless.tasks] Executing plugin WorkflowTriggerPlugin

[2024-02-19 06:58:24,341] [INFO] [paperless.tasks] WorkflowTriggerPlugin completed with no message

[2024-02-19 06:58:24,387] [INFO] [paperless.consumer] Consuming image.pdf

[2024-02-19 06:58:24,396] [DEBUG] [paperless.consumer] Detected mime type: application/pdf

[2024-02-19 06:58:24,407] [DEBUG] [paperless.consumer] Parser: RasterisedDocumentParser

[2024-02-19 06:58:24,412] [DEBUG] [paperless.consumer] Parsing image.pdf...

[2024-02-19 06:58:24,655] [DEBUG] [paperless.parsing.tesseract] Calling OCRmyPDF with args: {'input_file': PosixPath('/tmp/paperless/paperless-ngxfdr96a0q/image.pdf'), 'output_file': PosixPath('/tmp/paperless/paperless-cbv7dbcg/archive.pdf'), 'use_threads': True, 'jobs': 4, 'language': 'deu', 'output_type': 'pdfa', 'progress_bar': False, 'color_conversion_strategy': 'RGB', 'redo_ocr': True, 'clean': True, 'rotate_pages': True, 'rotate_pages_threshold': 12.0, 'sidecar': PosixPath('/tmp/paperless/paperless-cbv7dbcg/sidecar.txt')}

[2024-02-19 06:59:38,158] [DEBUG] [paperless.consumer] Generating thumbnail for image.pdf...

[2024-02-19 06:59:38,167] [DEBUG] [paperless.parsing] Execute: convert -density 300 -scale 500x5000> -alpha remove -strip -auto-orient -define pdf:use-cropbox=true /tmp/paperless/paperless-cbv7dbcg/archive.pdf[0] /tmp/paperless/paperless-cbv7dbcg/convert.webp

[2024-02-19 06:59:40,784] [DEBUG] [paperless.consumer] Saving record to database

[2024-02-19 06:59:40,784] [DEBUG] [paperless.consumer] Creation date from parse_date: 2023-08-04 00:00:00+02:00

[2024-02-19 06:59:41,275] [INFO] [paperless.handlers] Assigning correspondent MHD to 2023-08-04 iJames Internet-Fax

[2024-02-19 06:59:41,308] [INFO] [paperless.handlers] Assigning document type Bescheinigung to 2023-08-04 MHD iJames Internet-Fax

[2024-02-19 06:59:41,477] [DEBUG] [paperless.consumer] Deleting file /tmp/paperless/paperless-ngxfdr96a0q/image.pdf

[2024-02-19 06:59:41,485] [DEBUG] [paperless.parsing.tesseract] Deleting directory /tmp/paperless/paperless-cbv7dbcg

[2024-02-19 06:59:41,487] [INFO] [paperless.consumer] Document 2023-08-04 MHD iJames Internet-Fax consumption finished

[2024-02-19 06:59:42,953] [DEBUG] [paperless.classifier] Gathering data from database...

[2024-02-19 06:59:43,192] [INFO] [paperless.classifier] No updates since last training

[2024-02-19 06:59:43,196] [DEBUG] [paperless.tasks] Training data unchanged.

[2024-02-19 07:05:00,549] [DEBUG] [paperless.classifier] Gathering data from database...

[2024-02-19 07:05:00,787] [INFO] [paperless.classifier] No updates since last training

[2024-02-19 07:05:00,792] [DEBUG] [paperless.tasks] Training data unchanged.

[2024-02-19 08:05:00,602] [DEBUG] [paperless.classifier] Gathering data from database...

[2024-02-19 08:05:00,839] [INFO] [paperless.classifier] No updates since last training

[2024-02-19 08:05:00,844] [DEBUG] [paperless.tasks] Training data unchanged.

[2024-02-19 09:05:00,529] [DEBUG] [paperless.classifier] Gathering data from database...

[2024-02-19 09:05:00,792] [INFO] [paperless.classifier] No updates since last training

[2024-02-19 09:05:00,797] [DEBUG] [paperless.tasks] Training data unchanged.

[2024-02-19 10:05:00,514] [DEBUG] [paperless.classifier] Gathering data from database...

[2024-02-19 10:05:00,755] [INFO] [paperless.classifier] No updates since last training

[2024-02-19 10:05:00,774] [DEBUG] [paperless.tasks] Training data unchanged.

[2024-02-19 11:05:04,061] [DEBUG] [paperless.classifier] Gathering data from database...

[2024-02-19 11:05:04,303] [INFO] [paperless.classifier] No updates since last training
```

### Paperless-ngx version

2.5.3

### Host OS

Portainer/Docker on Debian

### Installation method

Docker - official image

### Browser

Firefox

### Configuration changes

Update to 2.5.3

### Please confirm the following

- [X] I believe this issue is a bug that affects all users of Paperless-ngx, not something specific to my installation.
- [X] I have already searched for relevant existing issues and discussions before opening this report.
- [X] I have updated the title field above with a concise description.

**Comment by shamoon** [comment]
To clarify, is the issue that the same file is being imported multiple times or that multiple files are generated from one file / email?

Providing a sample is usually helpful

**Comment by stumpylog** [comment]
I don't see any messages about duplicates here.  The files might look similar, but they aren't exact duplicates.  Some places may generate a document on the fly, making it appear the same, but not be identical.  We can't catch that.

But as above, more details and a sample are needed

**Comment by highTowerSU** [comment]
Hi,

i have scanned many documents per mail to paper@abc... They are stored in an IMAP-Mailbox. In the IMAP-Mailbox each scan is in one E-Mail with one PDF-Doc.

In the PDF-Doc are four pages, a PATCH-T - Seperator Page, one Page, a PATCH-T - Seperator Page, two pages and so on.

In other PDF-Docs the same routine.

The documents are send once, and get into paperless for about 5-7 times as single documents divided by the patch-t pages.

I have attached a example Page, with the corresponding results. With out changing anything, this time everything worked fine.... The main difference ist the amount of scanned pages.

Logs:
[2024-02-19 21:05:04,090] [DEBUG] [paperless.tasks] Training data unchanged.

[2024-02-19 22:05:00,487] [DEBUG] [paperless.classifier] Gathering data from database...

[2024-02-19 22:05:00,720] [INFO] [paperless.classifier] No updates since last training

[2024-02-19 22:05:00,724] [DEBUG] [paperless.tasks] Training data unchanged.

[2024-02-19 22:35:04,470] [DEBUG] [paperless.tasks] Skipping plugin CollatePlugin

[2024-02-19 22:35:04,470] [DEBUG] [paperless.tasks] Executing plugin BarcodePlugin

[2024-02-19 22:35:04,471] [DEBUG] [paperless.barcodes] Scanning for barcodes using PYZBAR

[2024-02-19 22:35:07,986] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 22:35:10,115] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 22:35:12,290] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 22:35:14,407] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 22:35:16,364] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 22:35:18,306] [DEBUG] [paperless.barcodes] Barcode of type CODE39 found: PATCHT

[2024-02-19 22:35:18,308] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 22:35:20,068] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 22:35:21,944] [DEBUG] [paperless.barcodes] Barcode of type CODE39 found: PATCHT

[2024-02-19 22:35:21,946] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 22:35:23,928] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 22:35:25,741] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 22:35:27,505] [DEBUG] [paperless.barcodes] Barcode of type CODE39 found: PATCHT

[2024-02-19 22:35:27,508] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 22:35:29,900] [DEBUG] [paperless.barcodes] Starting new document at idx 4

[2024-02-19 22:35:29,901] [DEBUG] [paperless.barcodes] Starting new document at idx 6

[2024-02-19 22:35:29,901] [DEBUG] [paperless.barcodes] Starting new document at idx 9

[2024-02-19 22:35:29,904] [DEBUG] [paperless.barcodes] Split into 4 new documents

[2024-02-19 22:35:29,918] [DEBUG] [paperless.barcodes] pdf no:0 has 4 pages

[2024-02-19 22:35:29,940] [DEBUG] [paperless.barcodes] pdf no:1 has 1 pages

[2024-02-19 22:35:29,943] [DEBUG] [paperless.barcodes] pdf no:2 has 2 pages

[2024-02-19 22:35:29,948] [DEBUG] [paperless.barcodes] pdf no:3 has 1 pages

[2024-02-19 22:35:30,014] [INFO] [paperless.barcodes] Created new task 57ca1b8d-a695-4dde-aedc-5b36a4ed4a15 for image_document_0.pdf

[2024-02-19 22:35:30,029] [INFO] [paperless.barcodes] Created new task b393ff0d-a0a8-484e-93a7-1621ee6a0a1c for image_document_1.pdf

[2024-02-19 22:35:30,045] [INFO] [paperless.barcodes] Created new task b3e94768-2086-4734-80db-5489806a9a86 for image_document_2.pdf

[2024-02-19 22:35:30,057] [INFO] [paperless.barcodes] Created new task 264f23a7-7761-4683-b2f8-06941f44d30f for image_document_3.pdf

[2024-02-19 22:35:30,069] [INFO] [paperless.tasks] BarcodePlugin requested task exit: Barcode splitting complete!

[2024-02-19 22:35:30,565] [DEBUG] [paperless.tasks] Skipping plugin CollatePlugin

[2024-02-19 22:35:30,565] [DEBUG] [paperless.tasks] Executing plugin BarcodePlugin

[2024-02-19 22:35:30,566] [DEBUG] [paperless.barcodes] Scanning for barcodes using PYZBAR

[2024-02-19 22:35:31,765] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 22:35:33,808] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 22:35:35,970] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 22:35:38,103] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 22:35:39,992] [DEBUG] [paperless.barcodes] Scanning for barcodes using PYZBAR

[2024-02-19 22:35:41,194] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 22:35:43,199] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 22:35:45,214] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 22:35:47,296] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 22:35:49,071] [DEBUG] [paperless.barcodes] Scanning for barcodes using PYZBAR

[2024-02-19 22:35:50,171] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 22:35:52,153] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 22:35:54,179] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 22:35:56,254] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 22:35:58,113] [INFO] [paperless.tasks] BarcodePlugin completed with no message

[2024-02-19 22:35:58,171] [DEBUG] [paperless.tasks] Executing plugin WorkflowTriggerPlugin

[2024-02-19 22:35:58,178] [INFO] [paperless.tasks] WorkflowTriggerPlugin completed with no message

[2024-02-19 22:35:58,207] [INFO] [paperless.consumer] Consuming image.pdf

[2024-02-19 22:35:58,214] [DEBUG] [paperless.consumer] Detected mime type: application/pdf

[2024-02-19 22:35:58,226] [DEBUG] [paperless.consumer] Parser: RasterisedDocumentParser

[2024-02-19 22:35:58,231] [DEBUG] [paperless.consumer] Parsing image.pdf...

[2024-02-19 22:35:58,558] [DEBUG] [paperless.parsing.tesseract] Calling OCRmyPDF with args: {'input_file': PosixPath('/tmp/paperless/paperless-ngxjl3bp9gb/image.pdf'), 'output_file': PosixPath('/tmp/paperless/paperless-xbtg1sz5/archive.pdf'), 'use_threads': True, 'jobs': 4, 'language': 'deu', 'output_type': 'pdfa', 'progress_bar': False, 'color_conversion_strategy': 'RGB', 'redo_ocr': True, 'clean': True, 'rotate_pages': True, 'rotate_pages_threshold': 12.0, 'sidecar': PosixPath('/tmp/paperless/paperless-xbtg1sz5/sidecar.txt')}

[2024-02-19 22:36:30,245] [DEBUG] [paperless.consumer] Generating thumbnail for image.pdf...

[2024-02-19 22:36:30,254] [DEBUG] [paperless.parsing] Execute: convert -density 300 -scale 500x5000> -alpha remove -strip -auto-orient -define pdf:use-cropbox=true /tmp/paperless/paperless-xbtg1sz5/archive.pdf[0] /tmp/paperless/paperless-xbtg1sz5/convert.webp

[2024-02-19 22:36:35,737] [DEBUG] [paperless.consumer] Saving record to database

[2024-02-19 22:36:35,738] [DEBUG] [paperless.consumer] Creation date from parse_date: 2024-01-22 00:00:00+01:00

[2024-02-19 22:36:36,409] [INFO] [paperless.handlers] Assigning correspondent BfA to 2024-01-22 iJames Internet-Fax

[2024-02-19 22:36:36,427] [INFO] [paperless.handlers] Assigning document type Information to 2024-01-22 BfA iJames Internet-Fax

[2024-02-19 22:36:36,565] [DEBUG] [paperless.consumer] Deleting file /tmp/paperless/paperless-ngxjl3bp9gb/image.pdf

[2024-02-19 22:36:36,575] [DEBUG] [paperless.parsing.tesseract] Deleting directory /tmp/paperless/paperless-xbtg1sz5

[2024-02-19 22:36:36,578] [INFO] [paperless.consumer] Document 2024-01-22 BfA iJames Internet-Fax consumption finished

[2024-02-19 22:36:36,962] [DEBUG] [paperless.tasks] Skipping plugin CollatePlugin

[2024-02-19 22:36:36,963] [DEBUG] [paperless.tasks] Executing plugin BarcodePlugin

[2024-02-19 22:36:36,963] [DEBUG] [paperless.barcodes] Scanning for barcodes using PYZBAR

[2024-02-19 22:36:37,323] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 22:36:39,132] [DEBUG] [paperless.barcodes] Scanning for barcodes using PYZBAR

[2024-02-19 22:36:39,468] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 22:36:41,210] [DEBUG] [paperless.barcodes] Scanning for barcodes using PYZBAR

[2024-02-19 22:36:41,547] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 22:36:43,260] [INFO] [paperless.tasks] BarcodePlugin completed with no message

[2024-02-19 22:36:43,272] [DEBUG] [paperless.tasks] Executing plugin WorkflowTriggerPlugin

[2024-02-19 22:36:43,275] [INFO] [paperless.tasks] WorkflowTriggerPlugin completed with no message

[2024-02-19 22:36:43,290] [INFO] [paperless.consumer] Consuming image.pdf

[2024-02-19 22:36:43,294] [DEBUG] [paperless.consumer] Detected mime type: application/pdf

[2024-02-19 22:36:43,302] [DEBUG] [paperless.consumer] Parser: RasterisedDocumentParser

[2024-02-19 22:36:43,306] [DEBUG] [paperless.consumer] Parsing image.pdf...

[2024-02-19 22:36:43,519] [DEBUG] [paperless.parsing.tesseract] Calling OCRmyPDF with args: {'input_file': PosixPath('/tmp/paperless/paperless-ngxo8mfzq24/image.pdf'), 'output_file': PosixPath('/tmp/paperless/paperless-rtqydt91/archive.pdf'), 'use_threads': True, 'jobs': 4, 'language': 'deu', 'output_type': 'pdfa', 'progress_bar': False, 'color_conversion_strategy': 'RGB', 'redo_ocr': True, 'clean': True, 'rotate_pages': True, 'rotate_pages_threshold': 12.0, 'sidecar': PosixPath('/tmp/paperless/paperless-rtqydt91/sidecar.txt')}

[2024-02-19 22:37:01,924] [DEBUG] [paperless.consumer] Generating thumbnail for image.pdf...

[2024-02-19 22:37:01,933] [DEBUG] [paperless.parsing] Execute: convert -density 300 -scale 500x5000> -alpha remove -strip -auto-orient -define pdf:use-cropbox=true /tmp/paperless/paperless-rtqydt91/archive.pdf[0] /tmp/paperless/paperless-rtqydt91/convert.webp

[2024-02-19 22:37:03,930] [DEBUG] [paperless.consumer] Saving record to database

[2024-02-19 22:37:03,930] [DEBUG] [paperless.consumer] Creation date from st_mtime: 2024-02-19 22:35:04.062931+01:00

[2024-02-19 22:37:04,591] [INFO] [paperless.handlers] Assigning correspondent BfA to 2024-02-19 iJames Internet-Fax

[2024-02-19 22:37:04,601] [INFO] [paperless.handlers] Assigning document type Vertrag to 2024-02-19 BfA iJames Internet-Fax

[2024-02-19 22:37:05,322] [DEBUG] [paperless.consumer] Deleting file /tmp/paperless/paperless-ngxo8mfzq24/image.pdf

[2024-02-19 22:37:05,329] [DEBUG] [paperless.parsing.tesseract] Deleting directory /tmp/paperless/paperless-rtqydt91

[2024-02-19 22:37:05,331] [INFO] [paperless.consumer] Document 2024-02-19 BfA iJames Internet-Fax consumption finished

[2024-02-19 22:37:05,690] [DEBUG] [paperless.tasks] Skipping plugin CollatePlugin

[2024-02-19 22:37:05,690] [DEBUG] [paperless.tasks] Executing plugin BarcodePlugin

[2024-02-19 22:37:05,691] [DEBUG] [paperless.barcodes] Scanning for barcodes using PYZBAR

[2024-02-19 22:37:06,206] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 22:37:08,164] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 22:37:09,851] [DEBUG] [paperless.barcodes] Scanning for barcodes using PYZBAR

[2024-02-19 22:37:10,402] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 22:37:12,430] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 22:37:14,045] [DEBUG] [paperless.barcodes] Scanning for barcodes using PYZBAR

[2024-02-19 22:37:14,566] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 22:37:16,569] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 22:37:18,342] [INFO] [paperless.tasks] BarcodePlugin completed with no message

[2024-02-19 22:37:18,368] [DEBUG] [paperless.tasks] Executing plugin WorkflowTriggerPlugin

[2024-02-19 22:37:18,371] [INFO] [paperless.tasks] WorkflowTriggerPlugin completed with no message

[2024-02-19 22:37:18,385] [INFO] [paperless.consumer] Consuming image.pdf

[2024-02-19 22:37:18,389] [DEBUG] [paperless.consumer] Detected mime type: application/pdf

[2024-02-19 22:37:18,397] [DEBUG] [paperless.consumer] Parser: RasterisedDocumentParser

[2024-02-19 22:37:18,401] [DEBUG] [paperless.consumer] Parsing image.pdf...

[2024-02-19 22:37:18,626] [DEBUG] [paperless.parsing.tesseract] Calling OCRmyPDF with args: {'input_file': PosixPath('/tmp/paperless/paperless-ngxedc9dfk4/image.pdf'), 'output_file': PosixPath('/tmp/paperless/paperless-8yq5kibh/archive.pdf'), 'use_threads': True, 'jobs': 4, 'language': 'deu', 'output_type': 'pdfa', 'progress_bar': False, 'color_conversion_strategy': 'RGB', 'redo_ocr': True, 'clean': True, 'rotate_pages': True, 'rotate_pages_threshold': 12.0, 'sidecar': PosixPath('/tmp/paperless/paperless-8yq5kibh/sidecar.txt')}

[2024-02-19 22:37:37,951] [DEBUG] [paperless.consumer] Generating thumbnail for image.pdf...

[2024-02-19 22:37:37,958] [DEBUG] [paperless.parsing] Execute: convert -density 300 -scale 500x5000> -alpha remove -strip -auto-orient -define pdf:use-cropbox=true /tmp/paperless/paperless-8yq5kibh/archive.pdf[0] /tmp/paperless/paperless-8yq5kibh/convert.webp

[2024-02-19 22:37:40,009] [DEBUG] [paperless.consumer] Saving record to database

[2024-02-19 22:37:40,010] [DEBUG] [paperless.consumer] Creation date from st_mtime: 2024-02-19 22:35:04.062931+01:00

[2024-02-19 22:37:40,646] [INFO] [paperless.handlers] Assigning correspondent Okai Vehicle Limited to 2024-02-19 iJames Internet-Fax

[2024-02-19 22:37:40,657] [INFO] [paperless.handlers] Assigning document type Bescheinigung to 2024-02-19 Okai Vehicle Limited iJames Internet-Fax

[2024-02-19 22:37:40,748] [DEBUG] [paperless.consumer] Deleting file /tmp/paperless/paperless-ngxedc9dfk4/image.pdf

[2024-02-19 22:37:40,763] [DEBUG] [paperless.parsing.tesseract] Deleting directory /tmp/paperless/paperless-8yq5kibh

[2024-02-19 22:37:40,765] [INFO] [paperless.consumer] Document 2024-02-19 Okai Vehicle Limited iJames Internet-Fax consumption finished

[2024-02-19 22:37:41,087] [DEBUG] [paperless.tasks] Skipping plugin CollatePlugin

[2024-02-19 22:37:41,088] [DEBUG] [paperless.tasks] Executing plugin BarcodePlugin

[2024-02-19 22:37:41,088] [DEBUG] [paperless.barcodes] Scanning for barcodes using PYZBAR

[2024-02-19 22:37:41,379] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 22:37:43,654] [DEBUG] [paperless.barcodes] Scanning for barcodes using PYZBAR

[2024-02-19 22:37:43,938] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 22:37:46,144] [DEBUG] [paperless.barcodes] Scanning for barcodes using PYZBAR

[2024-02-19 22:37:46,436] [DEBUG] [paperless.barcodes] Upscaling image by 2.0 for better barcode detection

[2024-02-19 22:37:48,654] [INFO] [paperless.tasks] BarcodePlugin completed with no message

[2024-02-19 22:37:48,668] [DEBUG] [paperless.tasks] Executing plugin WorkflowTriggerPlugin

[2024-02-19 22:37:48,671] [INFO] [paperless.tasks] WorkflowTriggerPlugin completed with no message

[2024-02-19 22:37:48,685] [INFO] [paperless.consumer] Consuming image.pdf

[2024-02-19 22:37:48,688] [DEBUG] [paperless.consumer] Detected mime type: application/pdf

[2024-02-19 22:37:48,696] [DEBUG] [paperless.consumer] Parser: RasterisedDocumentParser

[2024-02-19 22:37:48,700] [DEBUG] [paperless.consumer] Parsing image.pdf...

[2024-02-19 22:37:48,906] [DEBUG] [paperless.parsing.tesseract] Calling OCRmyPDF with args: {'input_file': PosixPath('/tmp/paperless/paperless-ngxmnsf8a0h/image.pdf'), 'output_file': PosixPath('/tmp/paperless/paperless-lyt_uha7/archive.pdf'), 'use_threads': True, 'jobs': 4, 'language': 'deu', 'output_type': 'pdfa', 'progress_bar': False, 'color_conversion_strategy': 'RGB', 'redo_ocr': True, 'clean': True, 'rotate_pages': True, 'rotate_pages_threshold': 12.0, 'sidecar': PosixPath('/tmp/paperless/paperless-lyt_uha7/sidecar.txt')}

[2024-02-19 22:38:04,613] [DEBUG] [paperless.consumer] Generating thumbnail for image.pdf...

[2024-02-19 22:38:04,618] [DEBUG] [paperless.parsing] Execute: convert -density 300 -scale 500x5000> -alpha remove -strip -auto-orient -define pdf:use-cropbox=true /tmp/paperless/paperless-lyt_uha7/archive.pdf[0] /tmp/paperless/paperless-lyt_uha7/convert.webp

[2024-02-19 22:38:07,038] [DEBUG] [paperless.consumer] Saving record to database

[2024-02-19 22:38:07,039] [DEBUG] [paperless.consumer] Creation date from st_mtime: 2024-02-19 22:35:04.062931+01:00

[2024-02-19 22:38:07,592] [INFO] [paperless.handlers] Assigning correspondent KSK to 2024-02-19 iJames Internet-Fax

[2024-02-19 22:38:07,597] [INFO] [paperless.handlers] Assigning document type Fahrzeugdokumente to 2024-02-19 KSK iJames Internet-Fax

[2024-02-19 22:38:07,658] [DEBUG] [paperless.consumer] Deleting file /tmp/paperless/paperless-ngxmnsf8a0h/image.pdf

[2024-02-19 22:38:07,664] [DEBUG] [paperless.parsing.tesseract] Deleting directory /tmp/paperless/paperless-lyt_uha7

[2024-02-19 22:38:07,666] [INFO] [paperless.consumer] Document 2024-02-19 KSK iJames Internet-Fax consumption finished





[2024-02-19 KSK iJames Internet-Fax.pdf](https://github.com/paperless-ngx/paperless-ngx/files/14337137/2024-02-19.KSK.iJames.Internet-Fax.pdf)
[2024-02-19 OVL iJames Internet-Fax.pdf](https://github.com/paperless-ngx/paperless-ngx/files/14337138/2024-02-19.OVL.iJames.Internet-Fax.pdf)
[2024-02-19 BA iJames Internet-Fax.pdf](https://github.com/paperless-ngx/paperless-ngx/files/14337139/2024-02-19.BA.iJames.Internet-Fax.pdf)
[image-1.pdf](https://github.com/paperless-ngx/paperless-ngx/files/14337147/image-1.pdf)

**Comment by highTowerSU** [comment]
[2024-01-22 BA iJames Internet-Fax.pdf](https://github.com/paperless-ngx/paperless-ngx/files/14337196/2024-01-22.BA.iJames.Internet-Fax.pdf)

forgotten File...

**Comment by highTowerSU** [comment]
Maybe the main Problem is, that one Mail with 30 pages in one pdf without any barcode leads to 7 duplicate entrys. This time they are filtered correctly, but even that should not happen. Why is one mail 8 times in the queue?

![grafik](https://github.com/paperless-ngx/paperless-ngx/assets/3651850/66d05c3a-ad68-4552-9dfc-f8fdbc18e65d)

**Comment by sbrunner** [comment]
I get the same issue as @highTowerSU ...

**Comment by highTowerSU** [comment]
I have scanned following E-Mails
![grafik](https://github.com/paperless-ngx/paperless-ngx/assets/3651850/3b8b5e7f-df13-4678-b602-d8dc875498e9)
![grafik](https://github.com/paperless-ngx/paperless-ngx/assets/3651850/420ef41c-b368-4045-a04e-64ec646a456b)

In the PDF-Docs attached to the mails one per mail are for Example four pages, a PATCH-T - Seperator Page, one Page, a PATCH-T - Seperator Page, two pages and so on.

they generated following Jobs:
![grafik](https://github.com/paperless-ngx/paperless-ngx/assets/3651850/6dae1824-c5b0-4ef2-90a9-f677ceb4e4ed)

And following new Documents...
![grafik](https://github.com/paperless-ngx/paperless-ngx/assets/3651850/88f2e390-741b-43b3-b411-6640ccdf64c3)

This ist the list of md5sums:
/media/documents/originals > md5sum 00003*
2d7a169483cbf38e2c3228500299a27f  0000300.pdf
201c46b0c0474b4575a6029980583382  0000301.pdf
b5fefbf9260cc7984e5c3208a73b5a7d  0000302.pdf
784d0f68b3b375fda9e27e570d21ce0c  0000303.pdf
47250650dbd088be21211e0bb00434e6  0000304.pdf
aabbd34f096de665b1d39335128b8c28  0000305.pdf
78f7c7e76b74de0f2d81e65eea48ac63  0000306.pdf
3f282e1975271efccb4e4be9e155dccf  0000307.pdf
f41d969791c0934713e7382c0df87fd7  0000308.pdf
bbad8b088b9df96995f98d1bec9edf73  0000309.pdf
03b32d1e5a6b2bf9e84ed43b0a7d7116  0000310.pdf
1d5102d4fda1dfee5818154c2923fdda  0000311.pdf
e00189897e91b6d7a2041ad28f9e7164  0000312.pdf
2fcdc21d63f9e387c8f0a5e6c9449ca2  0000313.pdf
7c130858c1b9e1fe0cb69f499ab287c9  0000314.pdf
43502138a333a02dd0c65645caf379c0  0000315.pdf
c7d46d28771ad3ca2538dadeddf78595  0000316.pdf
efcb2d21acaa2d11c536b599fe6fd17f  0000317.pdf
2312d8b73c8ccaa6fb444bc391bb7a30  0000318.pdf
9696d993fb73d52cdeb1e465ae409f93  0000319.pdf
19db683f64bb229712ab71fd8a76e8dd  0000320.pdf
c79f41ced1f2d18676f50b5c0a922042  0000321.pdf
d727ad1f462196af961e83eae5c473cc  0000322.pdf
b335a645cab9047fe7eb5863fc256687  0000323.pdf
54772403750ee20bc91b50d5805017c1  0000324.pdf
5ed096189b326ffcf2aabaa470feaa55  0000325.pdf
d14f554787f83d5cbeb67c620235b369  0000326.pdf
2bbe8a231f0eeb16d39d076eb303c926  0000327.pdf
507eacf55a5f889958c5e9eee2ce9410  0000328.pdf
be68d529f5b350433627f7d398a10ed5  0000329.pdf
b8a358047fd0c3aec46c3f4905841d70  0000330.pdf
35831352ca6b7f80a7d9007aaa6f6ae5  0000331.pdf
8c75af6edfbaafb31c7b6f1724f106e9  0000332.pdf
1e2b2907d0378f3915d16c2672623de7  0000333.pdf
eac9aae6c6144e84dc33bcc02cae274b  0000334.pdf
aaeda49c540d25f18efc2318aaf3c898  0000335.pdf
f2075d2de1ccc297892bedd610ed62b1  0000336.pdf
0fb2554bc2f6b9344d176db70375f065  0000337.pdf

**Comment by shamoon** [comment]
Our dev who is most familiar with this part of the software is AFK for the moment, thanks for your patience.

**Comment by highTowerSU** [comment]
Hi,

i think the real Problem ist based on reimporting the mail. The different MD5 IDs are based on splitting the pdfs based on PATCH-T.

What ID do you mean by unique email identifier? (Message-ID?). I am hosting my mails for my self in an standard Postfix + Dovecot Mail-Setup. In the Mailserver logs, this ID is only received once and has not changed over time.

How can i turn on Message-IDs in Debug logs? aka mail.log?

This mail should be there only once: Fwd: Digitale Versicherungsbescheinigung 123/456/789 from a@b.c

data/log/mail.log.6:[2024-02-20 22:45:32,330] [INFO] [paperless_mail] Rule paper-hightower.Paper: Consuming attachment 2024-02-20_Schriftwechsel_VERS_18-21.pdf from mail Fwd: Digitale Versicherungsbescheinigung 123/456/789 from a@b.c
data/log/mail.log.6:[2024-02-20 22:45:33,572] [INFO] [paperless_mail] Rule paper-hightower.Paper: Consuming attachment image.pdf from mail iJames Internet-Fax from brother-9142@a.b.c
data/log/mail.log.6:[2024-02-20 22:45:33,734] [INFO] [paperless_mail] Rule paper-hightower.Paper: Consuming attachment image.pdf from mail iJames Internet-Fax from brother-9142@a.b.c
data/log/mail.log.6:[2024-02-20 22:45:33,769] [INFO] [paperless_mail] Rule paper-hightower.Paper: Consuming attachment image.pdf from mail iJames Internet-Fax from brother-9142@a.b.c
data/log/mail.log.6:[2024-02-20 22:45:38,123] [INFO] [paperless_mail] Rule paper-hightower.Paper: Consuming attachment 2024-02-20_Schriftwechsel_VERS_18-21.pdf from mail Fwd: Digitale Versicherungsbescheinigung 123/456/789 from a@b.c
data/log/mail.log.6:[2024-02-20 22:45:39,315] [INFO] [paperless_mail] Rule paper-hightower.Paper: Consuming attachment image.pdf from mail iJames Internet-Fax from brother-9142@a.b.c
data/log/mail.log.6:[2024-02-20 22:45:39,480] [INFO] [paperless_mail] Rule paper-hightower.Paper: Consuming attachment image.pdf from mail iJames Internet-Fax from brother-9142@a.b.c
data/log/mail.log.6:[2024-02-20 22:45:39,523] [INFO] [paperless_mail] Rule paper-hightower.Paper: Consuming attachment image.pdf from mail iJames Internet-Fax from brother-9142@a.b.c
data/log/mail.log.6:[2024-02-20 22:45:43,363] [INFO] [paperless_mail] Rule paper-hightower.Paper: Consuming attachment 2024-02-20_Schriftwechsel_VERS_18-21.pdf from mail Fwd: Digitale Versicherungsbescheinigung 123/456/789 from a@b.c
data/log/mail.log.6:[2024-02-20 22:45:44,536] [INFO] [paperless_mail] Rule paper-hightower.Paper: Consuming attachment image.pdf from mail iJames Internet-Fax from brother-9142@a.b.c
data/log/mail.log.6:[2024-02-20 22:45:44,710] [INFO] [paperless_mail] Rule paper-hightower.Paper: Consuming attachment image.pdf from mail iJames Internet-Fax from brother-9142@a.b.c
data/log/mail.log.6:[2024-02-20 22:45:44,749] [INFO] [paperless_mail] Rule paper-hightower.Paper: Consuming attachment image.pdf from mail iJames Internet-Fax from brother-9142@a.b.c
data/log/mail.log.6:[2024-02-20 22:45:48,970] [INFO] [paperless_mail] Rule paper-hightower.Paper: Consuming attachment 2024-02-20_Schriftwechsel_VERS_18-21.pdf from mail Fwd: Digitale Versicherungsbescheinigung 123/456/789 from a@b.c
data/log/mail.log.6:[2024-02-20 22:45:50,173] [INFO] [paperless_mail] Rule paper-hightower.Paper: Consuming attachment image.pdf from mail iJames Internet-Fax from brother-9142@a.b.c
data/log/mail.log.6:[2024-02-20 22:45:50,348] [INFO] [paperless_mail] Rule paper-hightower.Paper: Consuming attachment image.pdf from mail iJames Internet-Fax from brother-9142@a.b.c
data/log/mail.log.6:[2024-02-20 22:45:50,386] [INFO] [paperless_mail] Rule paper-hightower.Paper: Consuming attachment image.pdf from mail iJames Internet-Fax from brother-9142@a.b.c
data/log/mail.log.6:[2024-02-20 22:45:54,245] [INFO] [paperless_mail] Rule paper-hightower.Paper: Consuming attachment 2024-02-20_Schriftwechsel_VERS_18-21.pdf from mail Fwd: Digitale Versicherungsbescheinigung 123/456/789 from a@b.c
data/log/mail.log.6:[2024-02-20 22:45:55,437] [INFO] [paperless_mail] Rule paper-hightower.Paper: Consuming attachment image.pdf from mail iJames Internet-Fax from brother-9142@a.b.c
data/log/mail.log.6:[2024-02-20 22:45:55,600] [INFO] [paperless_mail] Rule paper-hightower.Paper: Consuming attachment image.pdf from mail iJames Internet-Fax from brother-9142@a.b.c
data/log/mail.log.6:[2024-02-20 22:45:55,644] [INFO] [paperless_mail] Rule paper-hightower.Paper: Consuming attachment image.pdf from mail iJames Internet-Fax from brother-9142@a.b.c
data/log/mail.log.6:[2024-02-20 22:45:59,471] [INFO] [paperless_mail] Rule paper-hightower.Paper: Consuming attachment 2024-02-20_Schriftwechsel_VERS_18-21.pdf from mail Fwd: Digitale Versicherungsbescheinigung 123/456/789 from a@b.c
data/log/mail.log.6:[2024-02-20 22:46:00,627] [INFO] [paperless_mail] Rule paper-hightower.Paper: Consuming attachment image.pdf from mail iJames Internet-Fax from brother-9142@a.b.c
data/log/mail.log.6:[2024-02-20 22:46:00,956] [INFO] [paperless_mail] Rule paper-hightower.Paper: Consuming attachment image.pdf from mail iJames Internet-Fax from brother-9142@a.b.c
data/log/mail.log.6:[2024-02-20 22:46:01,013] [INFO] [paperless_mail] Rule paper-hightower.Paper: Consuming attachment image.pdf from mail iJames Internet-Fax from brother-9142@a.b.c
data/log/mail.log.6:[2024-02-20 22:46:04,858] [INFO] [paperless_mail] Rule paper-hightower.Paper: Consuming attachment 2024-02-20_Schriftwechsel_VERS_18-21.pdf from mail Fwd: Digitale Versicherungsbescheinigung 123/456/789 from a@b.c
data/log/mail.log.6:[2024-02-20 22:46:05,955] [INFO] [paperless_mail] Rule paper-hightower.Paper: Consuming attachment image.pdf from mail iJames Internet-Fax from brother-9142@a.b.c
data/log/mail.log.6:[2024-02-20 22:46:06,113] [INFO] [paperless_mail] Rule paper-hightower.Paper: Consuming attachment image.pdf from mail iJames Internet-Fax from brother-9142@a.b.c
data/log/mail.log.6:[2024-02-20 22:46:06,158] [INFO] [paperless_mail] Rule paper-hightower.Paper: Consuming attachment image.pdf from mail iJames Internet-Fax from brother-9142@a.b.c

**Comment by stumpylog** [comment]
UID is the unique identifier for the message, as specified in the IMAP v4 RFC.

An email will not be processed again if it already has been:
https://github.com/paperless-ngx/paperless-ngx/blob/d2f9b5d5e505bc724484bff1491e63863d0498c5/src/paperless_mail/mail.py#L583-L589

You can view the saved ProcessedMail in the admin pages.  I also notice the timing, you appear to either be calling or have configured the mail fetching to run very often.  An attachment must be completed to be skipped again.
